src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -688,9 +689,11 @@ ReactDOMComponent.Mixin = {
           }
           propValue = CSSPropertyOperations.createMarkupForStyles(propValue);
         }
-        var markup = null;
+        var markup = '';
         if (this._tag != null && isCustomComponent(this._tag, props)) {
-          markup = DOMPropertyOperations.createMarkupForCustomAttribute(propKey, propValue);
+          if (propKey != CHILDREN || typeof propValue === 'string' || typeof propValue === 'number') {
```

Comment:
`!==` (looks like we might have relaxed that when switching to fbjs lint rules)

@spicyj's long line length detector is slipping…